content/sql/concepts/math-functions/terms/asin/asin.md

```diff
@@ -0,0 +1,54 @@
+---
+Title: 'ASIN()'
+Description: 'Returns the arc sine of a number.'
+Subjects:
+  - 'Data Science'
+  - 'Computer Science'
+Tags:
+  - 'SQLite'
+  - 'PostgreSQL'
+  - 'MySQL'
+  - 'Functions'
+CatalogContent:
+  - 'learn-sql'
+  - 'paths/analyze-data-with-sql'
+---
+
+The **`ASIN()`** function in SQL is a mathematical function that returns the sine of an angle.  
+
+## Syntax
+
+```sql
+SELECT ASIN(number);
+```
```

Comment:
````suggestion
```pseudo
ASIN(number)
```
````
SELECT is not part of the syntax, and we generally place syntax blocks in a ` ```pseudo ` block.